torch/csrc/autograd/python_variable.cpp

```diff
@@ -1548,7 +1559,17 @@ void concrete_dispatch_fn(const c10::impl::PyInterpreter*, const c10::OperatorHa
         }
       }
     }
-    PyTuple_SET_ITEM(args.ptr(), idx, torch::jit::toPyObject(std::move(ivalue)).release().ptr());
+  }
+
+  // Populate positional arguments
+  for (int64_t idx = 0; idx < positional_end; idx++) {
```

Comment:
This is not folded in the loop above for clarity?
Because you could have it there if an `if (idx < positional_end)` and these two cases?